src/components/02_atoms/Widgets/EntityReferenceAutocomplete.js

```diff
@@ -147,13 +151,30 @@ class EntityReferenceAutocomplete extends React.Component {
       );
   };
 
+  cannotAddMoreEntities = () => {
```

Comment:
This is very nitpicky but the name of this function sounds a bit strange to me. How about just changing this to `getMaxItems` and then doing the comparison in the calling code?

Reply:
Sure, although the function does two checks, if is single and has an item, or if the max item has been entered. How about changing it to maxItemsSelected?

Reply:
I like @lauriii's idea, given we might want to render some description like: "N items left"

Reply:
I have updated the method, I used the same method from the file upload widget to check the maxItems.